Library/Homebrew/tap.rb

```diff
@@ -284,8 +284,8 @@ def install(options = {})
 
     link_completions_and_manpages
 
-    formula_count = formula_files.size
-    puts "Tapped #{Formatter.pluralize(formula_count, "formula")} (#{path.abv})" unless quiet
+    formatted_contents = Formatter.enumeration(*contents)&.prepend(" ")
```

Comment:
The `&.prepend(" ")` is neat; I wonder if it's worth making this an optional parameter for `Formatter.enumeration`